packages/uicore/src/components/Stepper/Stepper.stories.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React from 'react'
+import type { Meta, Story } from '@storybook/react'
+import { Title, Subtitle, ArgsTable, Stories, PRIMARY_STORY, Primary } from '@storybook/addon-docs/blocks'
+import { Stepper } from './Stepper'
+import { StepperProps } from './Stepper.types'
+import { StepList } from './__tests__/Stepper.mock'
+import { omit } from 'lodash-es'
+
+export default {
+  title: 'Components / Stepper',
+
+  component: Stepper,
+  parameters: {
+    docs: {
+      source: {
+        type: 'code'
+      },
+
+      page: function PageDescription() {
+        return (
+          <>
+            <Title>StepsProgress</Title>
+            <Subtitle>
+              <pre>
+                <code>{`import {Stepper} from '@harness/uicore'`}</code>
+              </pre>
+            </Subtitle>
+
+            <Primary />
+            <ArgsTable story={PRIMARY_STORY} />
+
+            <Stories />
+          </>
+        )
+      }
+    }
+  },
+  decorators: [Story => <Story />]
+} as Meta
+export const Basic: Story<StepperProps> = args => {
+  const { id = 'createSLOTabs', isStepValid = () => true, runValidationOnMount = false, stepList = StepList } = args
```

Comment:
should we give the id some genric default value?

Reply:
just mock data for stories , should be fine